iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -233,32 +239,43 @@ public enum ErrorCode
         DeviceMessageLockLost = 412002,
 
         // RequestEntityTooLarge - 413
```

Comment:
Copying elsewhere where we have a newline after a section header and enum values.